website/docs/components/button/partials/guidelines/guidelines.md

```diff
@@ -93,6 +93,11 @@ Buttons used for a Dropdown (with the chevron icon) can be found in [Dropdown](/
   <Hds::Button @color="secondary" @text="Cancel" />
 </Hds::ButtonSet>
 
+!!! Info
```

Comment:
There are already three banners in this page, we risk to water down their "visibility" (banner blindness). For me it works well just as a simple paragraph of text.